hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -387,68 +388,91 @@ if (fqtn != null && master.isInitialized()) {
     }
   }
 %>
-<table class="table table-striped">
-  <tr>
-    <th>RegionName</th>
-    <th>Start Key</th>
-    <th>End Key</th>
-    <th>Replica ID</th>
-    <th>RegionState</th>
-    <th>ServerName</th>
-  </tr>
-<%
-  final boolean metaScanHasMore;
-  byte[] lastRow = null;
-  try (final MetaBrowser.Results results = metaBrowser.getResults()) {
-    for (final RegionReplicaInfo regionReplicaInfo : results) {
-      lastRow = Optional.ofNullable(regionReplicaInfo)
-        .map(RegionReplicaInfo::getRow)
-        .orElse(null);
-      if (regionReplicaInfo == null) {
-%>
-  <tr>
-    <td colspan="6">Null result</td>
-  </tr>
-<%
-      continue;
-    }
+<div style="overflow-x: auto">
+  <table class="table table-striped nowrap">
+    <tr>
+      <th>RegionName</th>
+      <th>Start Key</th>
+      <th>End Key</th>
+      <th>Replica ID</th>
+      <th>RegionState</th>
+      <th>ServerName</th>
+      <th>Sequence Number</th>
+      <th>Target ServerName</th>
+      <th>Merge RegionNames</th>
+      <th>SplitA</th>
+      <th>SplitB</th>
```

Comment:
How we do merge? Merge names can be more merge001, merge002... we print them all out in Merge RegionNames? If so, why split different?

Reply:
Yes merge is all columns matching regex "info:merge*" in hbase:meta. Do we need to know the column name? For "info:split*" I think we can merge A/B into one item as well. I'll make the change as discussion in the JIRA.